contracts/test-helpers/DayLimitMock.sol

```diff
@@ -0,0 +1,15 @@
+pragma solidity ^0.4.4;
+import "../DayLimit.sol";
+
+contract DayLimitMock is DayLimit {
+  uint public totalSpending;
+
+  function DayLimitMock(uint _value, address[] _owners, uint _required) DayLimit(_value) Shareable(_owners, _required) {
+    totalSpending = 0;
+  }
+
+  function attemptSpend(uint _value) external limitedDaily(_value) {
```

Comment:
```limitedDaily(_value)``` isn't this using the `_value` parameter from this function instead of the contract's `dailyLimit` set on constructor?